docs/topics/installing-vs-code-extension.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * docs/vsc-extension-guide/master.adoc
+
+
+[id="installing-vs-code-extension_{context}"]
+= Installing the {ProductShortName} extension for Visual Studio Code
+
+You can install the {ProductShortName} extension for Visual Studio Code (VS Code) on any platform that supports VS Code.
+
+.Prerequisites
+
+* 8 GB RAM
+* JDK 11 or later
+
+
+.Procedure
+
+. Set the environmental variable `JAVA_HOME` to `jdk11`:
++
+[source, terminal]
+----
+$ export JAVA_HOME=jdk11
+----
+
+. In VS Code, click the *Extensions* icon on the Activity bar to open the *Extensions* view.
+. Enter `{ProductName}` in the Search field.
+. Select the *Migration Toolkit for Applications* extension and click *Install*.
++
+The {ProductShortName} extension icon (image:vs_MTA_extension_icon.png[MTA code extension]) is displayed on the Activity bar.
+
+[NOTE]
+====
+To uninstall the {ProductShortName} extension:
+
+. In the *Extensions* view, select the *{ProductName}* extension.
+. Click the *Manage* icon and select *Uninstall*.
```

Comment:
Is it called the Manage icon in the UI? Also, this is worth using an image of the icon. I think it's a gear.